odoo/orm/models.py

```diff
@@ -4259,8 +4280,14 @@ def create(self, vals_list: list[ValuesType]) -> Self:
                 if field.store:
                     stored[key] = val
                 if field.inherited:
+                    if not self._has_field_access(field, 'write'):
+                        # we cannot write the field, let it default when inheriting
+                        continue
```

Comment:
I don't get it :thinking:  ? The check of the `inherited` fields will be manage is already delegated later (next for loop block and manage by the inherits model).

Reply:
Same as for inherited fields.

Reply:
I can understand the code. But I will appreciate if there is an explicit test for it, even if some errors will be raised in other tests if we don't `continue` here.
we could test `the defualt value (not in context) for inherited fields won't be blocked by the field access check when create`